sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogDetailLevel.java

```diff
@@ -34,6 +39,36 @@
      */
     BODY_AND_HEADERS;
 
+    static final String BASIC_VALUE = "basic";
+    static final String HEADERS_VALUE = "headers";
+    static final String BODY_VALUE = "body";
+    static final String BODY_AND_HEADERS_VALUE = "body_and_headers";
+    static final String BODYANDHEADERS_VALUE = "bodyandheaders";
```

Comment:
I don't know that I would offer two variants?

Reply:
I went with both given the underscores in the enum to allow for both a more word like and a stricter enum like case.